.envrc

```diff
@@ -0,0 +1,7 @@
+if ! has nix_direnv_version || ! nix_direnv_version 3.0.6; then
+  source_url "https://raw.githubusercontent.com/nix-community/nix-direnv/3.0.6/direnvrc" "sha256-RYcUJaRMf8oF5LznDrlCXbkOQrywm0HDv1VjYGaJGdM="
+fi
+
+nix_direnv_manual_reload
+
+use flake .
```

Comment:
Can we throw this into another PR?

I like this idea, and do something similar on some personal project. Just prefer it to be it's own PR and not hidden in this one.